packages/app/src/cli/commands/app/function/run.ts

```diff
@@ -33,6 +33,11 @@ export default class FunctionRun extends AppUnlinkedCommand {
       description: 'Name of the WebAssembly export to invoke.',
       env: 'SHOPIFY_FLAG_EXPORT',
     }),
+    'invoked-by': Flags.string({
```

Comment:
this flag should have `hidden: true` as it is not intended to be used externally